### PR TITLE
E2e/feature/ri 3853 show process results for imported databases

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: tests/e2e/test-data/certs/client.key [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (ofersteinberg) [inline tests/e2e/test-data/certs/client.key:1]: I think you could make the whole certs in one file. But this is also a good one.

File: tests/e2e/test-data/rdm-full.json [hunk 2]
```diff
@@ -0,0 +1,219 @@
+[
```
Comment (ofersteinberg) [inline tests/e2e/test-data/rdm-full.json:1]: What is that file?
Please add a description on top

Comment (vlad-dargel) [reply tests/e2e/test-data/rdm-full.json:1]: no, it's json file

Comment (vlad-dargel) [reply tests/e2e/test-data/rdm-full.json:1]: which is only used to uploading this file in app

File: tests/e2e/test-data/certs/client.key (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/test-data/certs/client.key:1]: we decided to not put text to save a memory on CI (no matter what is inside this cert)

File: tests/e2e/pageObjects/browser-page.ts [hunk 3]
```diff
@@ -26,8 +26,6 @@ export class BrowserPage {
     setDeleteButton = Selector('[data-testid=set-delete-btn]');
     streamDeleteButton = Selector('[data-testid=stream-delete-btn]');
     myRedisDbIcon = Selector('[data-testid=my-redis-db-icon]');
-    streamDeleteButton = Selector('[data-testid=stream-delete-btn]');
```
Comment (nmammadli) [inline tests/e2e/pageObjects/browser-page.ts:29]: Why this selectors are deleted? We are using them on other tests

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**